src/main/java/com/sandvoxel/generitech/common/blocks/power/BlockGen.java

```diff
@@ -20,13 +19,13 @@
 
 import java.util.Random;
 
-public class BlockCables extends BlockMachineBase {
+public class BlockGen extends BlockMachineBase {
 
-    public BlockCables() {
+    public BlockGen() {
         super(Material.ROCK, "machines/pulverizer/pulverizer", MachineTier.all());
         this.setDefaultState(blockState.getBaseState().withProperty(MACHINETIER, MachineTier.TIER_0));
         this.setTileEntity(TestPower.class);
-        this.setCreativeTab(GeneriTechTabs.GENERAL);
+        //this.setCreativeTab(GeneriTechTabs.GENERAL);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3ACommentedOutCodeLine)